docs/source/usage.rst

```diff
@@ -231,6 +231,12 @@ You can provide default values directly in the accessing code:
     >>> conf.get('missing_key', 'a default value')
     'a default value'
 
+By default, `conf.get` returns `None` if the key is not found:
+.. doctest:: loaded
+
+    >>> conf.get('missing_key') is None
+    True
+
```

Comment:
I think since this is the behavior from dict we shouldn't document it.

Reply:
Ok